src/CAL-1.0.xml

```diff
@@ -0,0 +1,470 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SPDXLicenseCollection xmlns="http://www.spdx.org/license">
+  <license isOsiApproved="true" licenseId="CAL-1.0" name="Cryptographic Autonomy License 1.0">
+    <crossRefs>
+      <crossRef>http://cryptographicautonomylicense.com/license-text.html</crossRef>
+      <crossRef>https://opensource.org/licenses/CAL-1.0</crossRef>
+    </crossRefs>
+    <notes>The first draft of this license was released February 2019, and the fourth revision was approved by the OSI February 2020.</notes>
+    <text>
+      <titleText>
+        <p><optional># </optional>The Cryptographic Autonomy License, v. 1.0</p>
+      </titleText>
+        <p>
+<optional>*</optional>This Cryptographic Autonomy License (the "License") 
+applies to any Work whose owner has marked it with any of the following 
```

Comment:
The tests are probably failing due to the `<optional>*</optional>`.  There is a known bug with the tool that publishes the license where it doesn't deal with single characters being optional.  If this is the case, you can work around this by making it a variable text '<alt match="(\*This|This)" name="this">*This</alt>`